src/api/app/models/project.rb

```diff
@@ -608,7 +608,11 @@ def find_repos(sym)
   def update_instance(namespace = 'OBS', name = 'UpdateProject')
     # check if a newer instance exists in a defined update project
     a = find_attribute(namespace, name)
-    return Project.find_by_name(a.values[0].value) if a && a.values[0]
+    if a && a.values[0]
+      update_instance = Project.find_by_name(a.values[0].value)
+      return update_instance if update_instance
```

Comment:
Just add an empty line and [rubocop](https://app.circleci.com/pipelines/github/openSUSE/open-build-service/2242/workflows/4a54f33d-b4b5-434b-a23e-509c8a95547a/jobs/54477) will be happy
```suggestion
      return update_instance if update_instance
      
```